context/context.go

```diff
@@ -32,7 +32,7 @@ var (
 	tmsiGenerator                    *idgenerator.IDGenerator = nil
 	amfUeNGAPIDGenerator             *idgenerator.IDGenerator = nil
 	amfStatusSubscriptionIDGenerator *idgenerator.IDGenerator = nil
-	AmfContextMutex                  sync.Mutex
+	AmfContextMutex                  sync.RWMutex
```

Comment:
This is only used locally, so no need to exported, correct?
```suggestion
	amfContextMutex                  sync.RWMutex
```
Also, see my comment about using `Mutex` or `RWMutex`